samples/subsys/usb/audio/headset/sample.yaml

```diff
@@ -1,2 +1,7 @@
 sample:
-  name: USB audio sample
+  name: USB audio headset sample
```

Comment:
Let's make it consistent for both samples and start with capital letter as well.